src/sshping.cxx

```diff
@@ -346,7 +445,7 @@ ssh_session begin_session() {
     }
 
     // Try to connect
-    clock_gettime(CLOCK_MONOTONIC, &t0);
+	t0 = get_time();
```

Comment:
1 tabs to 4 spaces

May not be a bad idea to put a .clang_format file in this repo